CHANGELOG.md

```diff
@@ -2,6 +2,85 @@
 
 This document outlines major changes between releases.
 
+## 0.99.5 "Underestimation" (11 Nov 2022)
+
+It wasn't long since the previous release before a juicy set of features and
+optimisations came on board, so we've decided to change our release plan and
+supply one more 3.4.0 compatible version. It comes up with a massive suite
+of network P2P and broadcast improvements. We've carefully investigated message
+sending timeouts on benchmarks, reworked the broadcast logic and significantly
+reduced delays in message sending and amount of useless traffic between our
+nodes. For smart-contract developers and RPC client users several new features
+were added. These are automatic RPC bindings generator, poll-based and
+websocket-based transactions awaiting and extended web-socket client
+subscriptions interface. For better debugging experience we've improved state
+related functionality of VM CLI and added an ability to reset chain state to a
+particular height.
+
+An important dBFT initialization bug that caused timestamp-related errors on
+block addition was fixed. Also, those who use NeoGo node with
+EnableCORSWorkaround setting may want to update to be able to handle web-socket
```

Comment:
```suggestion
EnableCORSWorkaround setting may want an update to be able to handle web-socket
```

Reply:
Let it be the `may want to update the node`, is it OK?